samples/python/gemm_swiglu_mlp_fusion.py

```
@@ -0,0 +1,285 @@
+        for i in range(n):
+            try:
+                g.execute_plan_at_index(var_pack, ws, index=i, handle=handle)  # warm up / validity
+                torch.cuda.synchronize(dev)
+                start.record()
+                for _ in range(_AUTOTUNE_ITERS):
+                    g.execute_plan_at_index(var_pack, ws, index=i, handle=handle)
+                stop.record()
+                stop.synchronize()
+                times[i] = start.elapsed_time(stop) / _AUTOTUNE_ITERS
+            except Exception:
+                pass  # a plan may build but fail to execute on this shape; skip it
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)